Tests/LibSemVer/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+set(TEST_SOURCES
+    TestFromStringView.cpp
```

Comment:
How come the test cases here are not part of `TestSemVer`?

Reply:
What do you mean?

Reply:
Oh, you mean why I haven't clubbed these test cases to one file? That is because it is the namespace level function and TestSemVer contains test cases for the object methods. 

So to keep things simple and easy to understand, I had broken down these test-cases to two files.